src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -107,3 +143,21 @@ def get_helm_registry(profile, location):
     telemetry.set_exception(exception=str(response.json()), fault_type=consts.Get_HelmRegistery_Path_Fault_Type,
                             summary='Error while fetching helm chart registry path')
     raise CLIError("Error while fetching helm chart registry path: {}".format(str(response.json())))
+
+
+def get_flag_from_values_file(values_file):
+    isClientSecretATokenFlag = None
+    with open(values_file) as f:
+        try:
+            docs = yaml.safe_load(f)
+            if 'systemDefaultValues' not in docs:
+                raise CLIError("Incorrect values file format. Provide 'isClientSecretAToken' flag under 'systemDefaultValues' .yml file.")
+            else:
```

Comment:
else is not needed here. Add the remaining code below if because you are already failing the code in if

Reply:
okay